config/mappings.json

```diff
@@ -2,6 +2,7 @@
   "MyCloudantDB": {
     "searchPatterns": [
       "cloudfoundry:<cloudant instance name>",
+      "cloudfoundry:cloudant",
```

Comment:
@NTripleA I see two entries for cloudfoundy in the mappings.json file... there should be only one. If the new entry is now `cloudfoundry:cloudant` that implies that the developer must name the instance of the Cloudant db `cloudant`. Is that's the intention, the README should be updated accordingly. For instance, see `https://github.com/IBM-Bluemix/get-started-swift#6-use-the-database`.

Reply:
Yes, you are right. It does add redundancy. I'll delete it. And the purpose of writing ```cloudant``` is for it to have a default naming convention for the services. They can change it anytime they want, of course. But it's just to make it consistent with the other getting started apps. The README will be changed accordingly soon. But I have to work with one of the writers from the docs team on this. Because she wants to change various parts of it to make it consistent with the Liberty getting started app README.